dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.39/V2_39_1__Add_index_to_reserved_value.sql

```diff
@@ -0,0 +1 @@
+create unique index if not exists in_reservedvalue_value_generation on reservedvalue using btree (ownerobject, owneruid, key, lower (value));
```

Comment:
Since we are basically recreating an index we already have, lets remove the old one.